### DISCUSSION TITLE
Templates for Verified Credentials

### DISCUSSION THREAD

**Discussion by jayadinni** [Sunbird-RC/community#154] [Ideas & Enhancements]
Reference templates that issuers (who are using Sunbird RC) can configure:
Things considered:
1. Print version is A4 size as it can printed anywhere.
2. Scalable and accommodative to more or less number of labels. When the labels are very less, ensure that there are less white spaces.
3. For native version, the VC would include verifiable/QR code, and important values. 
4. Full version/Print version, will include all details configured.

Template options:
Print version:
![image](https://user-images.githubusercontent.com/580711/144090955-3df0e39f-692c-4e4a-9c7d-9f5f314a6e80.png)

Native/ version:
![image](https://user-images.githubusercontent.com/580711/144091884-05c50577-c045-417f-bd57-e3b64333f5c6.png)

Need inputs on:
1. Feasibility/Ability of issuer to choose and configure template.
2. Currently we have printable version of VC, where user can view and download. Can we have Digital/Native version too which is summary view and also works for most of the devices.

**Comment by pramodkvarma** [comment]
We discussed this. These layouts should be part of credential template configuration so that issuers can issue various types of VCs in their own templates. Software should NOT be opinionated about this. Give these as sample reference templates.
